src/controller/CHIPDeviceController.cpp

```diff
@@ -1562,10 +1555,6 @@ void DeviceCommissioner::OnDeviceConnectionFailureFn(void * context, PeerId peer
     {
         commissioner->CommissioningStageComplete(error);
     }
-    else
```

Comment:
I can't comment previous lines. In line 1550, there is a check of `commissioner->mPairingDelegate != nullptr`, since the field is no longer being used in this function, are we going to remove that check ?

Reply:
Oh, good catch!  That check was wrong in fact, because it skipped the the `ReleaseSession` cleanup!